src/vue/backend/src/App.vue

```diff
@@ -2,31 +2,36 @@
   <idx-block className="app">
     <idx-block className="nav">
```

Comment:
Lets not commit our test code so we can avoid merge conflicts with other components